bin/epaa.py

```diff
@@ -729,15 +729,22 @@ def create_affinity_values(allele, length, j, method, max_scores, allele_strings
         return np.nan
 
 
-def create_binder_values(aff, method):
+def create_binder_values(aff, method, thresholds):
     if not pd.isnull(aff):
-        if 'syf' in method:
-            return True if aff > 50.0 else False
-        else:
-            return True if aff <= 500.0 else False
+        return True if aff > thresholds.get(method) else False
```

Comment:
I think we still have to differentiate between the methods here since it is `>` or `<=`.

Reply:
Oh my bad, yes sure that needs to be considered. For NetMHC's Rank `>` needs to be added and for NetMHC's affinity `<=`. Correct me if I'm wrong.

Reply:
it should be `<` for rank, but let's add that when we have the functionality for it. For the affinity it has to be `<=`.